scripts/build-opentracing.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+yum install -y centos-release-scl epel-release
+
+yum install -y \
+        openresty-openssl-devel \
+        openresty-pcre-devel \
+        devtoolset-7 cmake3 pcre-devel \
+        openssl-devel thrift-devel \
+	GeoIP-devel systemtap-sdt-devel \
+	git libxml2-devel libxslt-devel \
+        gd-devel
+
+# Source the devtoolset-7, building tools (gcc...)
+# Sourced before the fail modes due to an unbound variable in the script
+# shellcheck disable=SC1091
+source scl_source enable devtoolset-7
+set -euo pipefail
+IFS=$'\n\t'
+
+OPENTRACING_CPP_VERSION=v1.2.0
+NGINX_OPENTRACING_VERSION=v0.2.1
+JAEGER_CPP_VERSION=v0.2.0
+OPENSSL_VERSION="OpenSSL_1_0_2k"
+OPENRESTY_MD5="637f82d0b36c74aec1c01bd3b8e0289c"
+
+curl --retry-delay 5 --retry 3 -s -L https://openresty.org/download/openresty-"${OPENRESTY_RPM_VERSION}".tar.gz -o /tmp/openresty.tar.gz
+md5sum -c <<<"${OPENRESTY_MD5} /tmp/openresty.tar.gz"
+tar zxf /tmp/openresty.tar.gz -C /tmp/
+
+git clone -b "$OPENTRACING_CPP_VERSION" https://github.com/opentracing/opentracing-cpp.git /tmp/opentracing
+git clone -b "$NGINX_OPENTRACING_VERSION" https://github.com/opentracing-contrib/nginx-opentracing.git /tmp/nginx-opentracing
+git clone -b "$JAEGER_CPP_VERSION" https://github.com/jaegertracing/cpp-client.git /tmp/jaeger-cpp
+git clone -b "${OPENSSL_VERSION}" https://github.com/openssl/openssl.git /tmp/openssl
+
+###################### fix
+export RESTY_PCRE_VERSION="8.41"
```

Comment:
Would `-devel` package of https://github.com/openresty/openresty-packaging/blob/master/rpm/SPECS/openresty-pcre.spec help?

Reply:
I see it is installed, so why do we need this?

Reply:
I will try to reuse those one.

Reply:
It really needs the source code..